data-plane/src/lib.rs

```diff
@@ -154,71 +156,47 @@ impl From<ProvisionerContext> for CageContext {
     }
 }
 
+#[derive(Clone, Deserialize)]
+pub struct FeatureContext {
+    pub api_key_auth: bool,
+    pub healthcheck: Option<String>,
+    pub trx_logging_enabled: bool,
+    #[serde(default)]
```

Comment:
Were these defaults a backwards compat thing? just noticed trx_logging_enabled doesn't have it

Reply:
Yep probably were, can remove them assuming they're no longer optional starting v1 of the CLI

Reply:
discussed offline — v1 of the CLI will make all of these fields required, so defaults can be removed.